### PR TITLE
{bio}[GCC/8.3.0] RNA-Bloom v1.2.3, minimap2 v2.17, ntCard v1.2.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2461.golett.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/13a56a847a427f003d14fcf0e9e13a00 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
generoso-1 - Linux centos linux 7.6.1810, x86_64, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/ce594f3e643e8bb6984b2cebef3f5219 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3163.skitty.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/af412cb48cc749c937d63d83e9720f46 for a full test report.

File: easybuild/easyconfigs/r/RNA-Bloom/RNA-Bloom-1.2.3-GCC-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+name = 'RNA-Bloom'
+version = '1.2.3'
+
+homepage = 'https://github.com/bcgsc/RNA-Bloom'
+description = "RNA-Bloom is a fast and memory-efficient de novo transcript sequence assembler."
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+source_urls = ['https://github.com/bcgsc/%(name)s/releases/download/v1.2.3']
+sources = ['rnabloom_v1.2.3.tar.gz']
```
Comment (Micket) [inline easybuild/easyconfigs/r/RNA-Bloom/RNA-Bloom-1.2.3-GCC-8.3.0.eb:12]:
```suggestion
sources = ['rnabloom_v%(version)s.tar.gz']
```

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/b40c3bf2011b22c0d93747c2a277da75 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/68021aec58889dac2a5daa26d18ff7b4 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**